app/src/main/java/org/mate/accessibility/check/widgetbased/ErrorMessageCheck.java

```diff
@@ -0,0 +1,54 @@
+package org.mate.accessibility.check.widgetbased;
+
+import org.mate.accessibility.AccessibilityViolation;
+import org.mate.accessibility.AccessibilityViolationTypes;
+import org.mate.state.IScreenState;
+import org.mate.ui.Widget;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ErrorMessageCheck implements IWidgetAccessibilityCheck {
+
+
+
+    public ErrorMessageCheck(){
```

Comment:
Empty public constructor is generated by default and does not have to be declared manually.

Reply:
right